src/render.jl

```diff
@@ -136,13 +175,13 @@ function relpath(path::ByteString, startpath::ByteString = ".")
     prefix_num = findlast(x -> !isempty(x), start_arr) - i - 1
     if prefix_num >= 0
         prefix = pardir * Base.path_separator
-        relpath_ = isempty(pathpart)                                      ?
-            (prefix^prefix_num) * pardir                                  :
-            (prefix^prefix_num) * pardir * Base.path_separator * pathpart
+        relpath_ = isempty(pathpart)                                          ?
+                (prefix^prefix_num) * pardir                                  :
+                (prefix^prefix_num) * pardir * Base.path_separator * pathpart
```

Comment:
4 space indenting with this rather than 8, but could probably be written as:

```julia
relpath = string(prefix^prefix_num, pardir, isempty(pathpart) ? "" : Base.path_separator * pathpart)
```

instead.